modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/trust/storage/TrustDbStorage.scala

```diff
@@ -0,0 +1,12 @@
+package org.tessellation.dag.l1.domain.trust.storage
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.TrustDbValues
+
+trait TrustDbStorage[F[_]] {
+  def getTrustScore(peerId: PeerId): F[Option[Double]]
```

Comment:
we prefer using nominal types https://github.com/estatico/scala-newtype for such values, to make it less error-prone (see transaction amount and fee)

Reply:
Converted `rating`, `score`, and `observationAdjustment` to `@newtype` value classes.